docker-integrations/tendermint/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Build Stage 1
```

Comment:
@luis-rf Can you add the following in the top of the Dockerfile?

```
# Use multi-stage builds to separate build-time and runtime dependencies
# This reduces the final image size
```

Also please include comments above every Dockerfile instruction as this Dockerfile will be our boilerplate for all runtimes, which is something we want to instruct users to use. Let's make it as much readable and cohesive as we can.